docs/source/llms/rag/notebooks/index.rst

```diff
@@ -14,6 +14,7 @@ the user's question.
     :hidden:
 
     question-generation-retrieval-evaluation.ipynb
+    retriever-evaluation-tutorial.ipynb
```

Comment:
The main nav card on the LLM landing page that links to this page will need to be updated. The text on that card says "Question Generation for RAG" with `<p>` tag text that talks solely about generating a question set for RAG. 

Can that card be updated to be more general about RAG applications?

https://github.com/mlflow/mlflow/blob/70e72373f494a39fed5d131ec053833a34bc3cdc/docs/source/llms/index.rst?plain=1#L382-L392

Reply:
The main landing page needs updating too :) 

https://github.com/mlflow/mlflow/blob/70e72373f494a39fed5d131ec053833a34bc3cdc/docs/source/index.rst?plain=1#L145-L155